packages/relay/test/report.test.ts

```diff
@@ -592,6 +593,150 @@ describe('POST /api/report', function () {
             })
     })
 
+    it('should settle report and update object status', async function () {
+        // insert mock value into report
+        const prevEpoch = await sync.loadCurrentEpoch()
+        db.update('ReportHistory', {
```

Comment:
update function is async function, therefore should wait for update function before following test